src/back/CrossTeamTestSuite/src/CrossTeamTestSuite/Data/Repository/Events/EventRepository.cs

```diff
@@ -0,0 +1,14 @@
+namespace CrossTeamTestSuite.Data.Repository.Events;
+
+public class EventRepository
+{
+    private List<Event> events = new();
+    public string DefaultName => "Company name: don't copy";
```

Comment:
Is this what you meant or you forgot to change that after copy?

Reply:
I meant it, I don't want to someone use this name for another organizer